README.md

```diff
@@ -1,6 +1,8 @@
 # Keepalived Exporter
 Prometheus exporter for [Keepalived](https://keepalived.org) metrics.
 
+[![Continuous Integration](https://github.com/cafebazaar/keepalived-exporter/workflows/Continuous%20Integration/badge.svg)](https://github.com/cafebazaar/keepalived-exporter/actions)
```

Comment:
Thanks!
I think it would be better to go under "Keepalived Exporter" title, upper than description.

Reply:
> Thanks!
> I think it would be better to go under "Keepalived Exporter" title, upper than description.

Agree )